controllers/kubeadmconfig_controller.go

```diff
@@ -121,12 +121,6 @@ func (r *KubeadmConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 	// Check for control plane ready. If it's not ready then we will requeue the machine until it is.
 	// The cluster-api machine controller set this value.
 	if cluster.Annotations[ControlPlaneReadyAnnotationKey] != "true" {
-		// if it's NOT a control plane machine, requeue
-		if !util.IsControlPlaneMachine(machine) {
-			log.Info("Control plane is not ready, requeing worker nodes until ready.")
-			return ctrl.Result{RequeueAfter: 30 * time.Second}, nil
-		}
-
```

Comment:
With removing this, should the log message on line 126 be updated as well to indicate that it will be requeueing both control plane and worker node machines?

Reply:
Ah ha, that log was copy pasta'd. Let me update that

Reply:
The comments were also not up-to-date so I took the liberty of updating those as well. The lock stuff is taken care of in #122